app/models/content_item.rb

```diff
@@ -51,6 +51,10 @@ def available_translations
     translations.sort_by { |t| t["locale"] == I18n.default_locale.to_s ? "" : t["locale"] }
   end
 
+  def slug
```

Comment:
Hmm, I'm very suspicious of anything related to slug, because it's so variable between models. If it's currently only being used in SimpleSmartAnswers, I think I'd advise adding it to that model and then we can bring it back here when we have a few other things that use it and we're sure that it is worth making a base attribute.

Reply:
Fair enough. 👍

Reply:
@KludgeKML I've made the change and fixed up.